config_templates/gretel/synthetics/low-record-count.yml

```diff
@@ -1,15 +1,7 @@
 # For datasets that have a low number of records, in the hundreds or so.
```

Comment:
Does this make the low-record-count.yml and default.yml essentially the same? Maybe add a backlog ticket to consider removing the low-record-count if they don't diverge in the medium-term future.

Reply:
The notable difference is a vocab size of 0 (zero) vs. 20k, which you'd really want for low record counts